thali/thalireplicationmanager.js

```diff
@@ -66,8 +67,8 @@ ThaliReplicationManager.prototype.start = function (deviceName, port, dbName) {
         this._emitter.addListener(NETWORK_CHANGED, networkChanged.bind(this));
         this.emit(ThaliReplicationManager.events.STARTED);
       }
-    }.bind(this));
-  }.bind(this));
+    }.bind(this)); //startBroadcasting
```

Comment:
Please remove the comment on line 70 and 71. This is just the kind of comment that rots really quickly as soon as there is an edit. We have IDEs to handle matching squiggly braces.